site/_shortcodes/Playlist.js

```diff
@@ -0,0 +1,210 @@
+const {html} = require('common-tags');
+const fetch = require('node-fetch/lib/index');
+const apiKey = 'AIzaSyAzOX6K8IXsxPwQ0rgW7QgJPndroPOiWfc';
+const maxResults = 50;
+const part = 'snippet';
```

Comment:
Use uppercase snake_case for config constants, like `API_KEY`, `MAX_RESULTS`, ...